Bottom: 215d6a276bce3e8fdef7194476d083c98699af61
Top:    8d08fbeb30ab915cfac4714d71272fa1134ce238
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:21:01 -0500

Add new host buildbot-worker.novatech-llc.com

Runs buildbot worker containers


---
diff --git a/ansible-playbook/production b/ansible-playbook/production
index bae6e4f9..1223c9db 100644
--- a/ansible-playbook/production
+++ b/ansible-playbook/production
@@ -6,6 +6,10 @@ daedalus.novatech-llc.com
 
 [docker-hosts]
 daedalus.novatech-llc.com
+builder.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
+
+[builder]
+builder.novatech-llc.com
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index ad0ef4ac..fa054a69 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -6,6 +6,10 @@ testdaedalus.novatech-llc.com
 
 [docker-hosts]
 testdaedalus.novatech-llc.com
+builder.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
+
+[builder]
+builder.novatech-llc.com
